crates/bevy_ui/Cargo.toml

```diff
@@ -27,6 +27,7 @@ bevy_sprite = { path = "../bevy_sprite", version = "0.9.0" }
 bevy_text = { path = "../bevy_text", version = "0.9.0" }
 bevy_transform = { path = "../bevy_transform", version = "0.9.0" }
 bevy_window = { path = "../bevy_window", version = "0.9.0" }
+bevy_winit = { path = "../bevy_winit", version = "0.9.0" }
```

Comment:
I find this interesting. This directly couples `bevy_ui` with `winit`. If we were to have other windowing solutions, `bevy_ui` would be incompatible, it also means that `bevy_ui` cannot be used without pulling `bevy_winit` with it.  

We may actually want this, but there's likely a discussion to be had about this coupling.

Reply:
`bevy_a11y` sounds good to me. We have first party crates that have less functionality.